examples/pytorch/gin/README.md

```diff
@@ -28,15 +28,22 @@ An experiment on the GIN in customized settings can be run with
 python main.py [--device 0 | --disable-cuda] --dataset COLLAB \
                --graph_pooling_type max --neighbor_pooling_type sum
 ```
+hint: **following the settings like the paper** is essential to reproduce the result reported.
```

Comment:
This line is not needed. You just need to clearly list the commands for different datasets in the "Results" section.